translation_x: 475361.340403
translation_y: 5525036.721478
orientation: 2.059036
utm_zone: '32U'
write_debug_file: True
max_covariance: 10.0
min_pose_distance: 0.2


